client/src/app/_services/auth.service.ts

```diff
@@ -56,28 +56,28 @@ export class AuthService {
       .pipe(tap(this.clearUser.bind(this)));
   }
 
-  definePassword(username: string, password: string, passwordToken: string, oldPassword: string): Observable<any> {
+  definePassword(email: string, password: String, passwordToken: String, oldPassword: String): Observable<any> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this 'String' wrapper object with primitive type 'string'. ([typescript:S1533](https://sonarcloud.io/organizations/k-fet/rules#rule_key=typescript%3AS1533))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=k-app&pullRequest=291&issues=AWU5BUYmOTG07BqbJp4Z&open=AWU5BUYmOTG07BqbJp4Z)

[ISSUE_KEY]: AWU5BUYmOTG07BqbJp4Z